g2o/core/optimization_algorithm_factory.h

```diff
@@ -24,8 +24,8 @@
 // NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#ifndef G2O_OPTMIZATION_ALGORITHM_PROPERTY_H
-#define G2O_OPTMIZATION_ALGORITHM_PROPERTY_H
+#ifndef G2O_OPTIMIZATION_ALGORITHM_FACTORY_H
```

Comment:
The name here referred to another header, but there was also a typo that made it unique.